File: Sum/Sum.py
# Sum of n numbers:

sum = 0
n = int(input("Enter number: "))
for i in range(n + 1):
    sum += i
print(f"Sum of {n} numbers is {sum}")


# Sum of square:

sum = 0
n = int(input("Enter number: "))
for i in range(n + 1):
    sum += i * i
    print(f"sqaure of {i} = {i*i} and sum = {sum}")
print(f"Sum of sqaure of {n} square  numbers is {sum}")

# Sum of cubes:

sum = 0
n = int(input("Enter number: "))
for i in range(n + 1):
    sum += i * i * i
    print(f"cube of {i} = {i*i*i} and sum = {sum}")
print(f"Sum of sqaure of {n} cube  numbers is {sum}")

File: Sum/Sum.cpp
// Sum of n numbers:

// #include <iostream>
// using namespace std;
// int main()
// {
//     int n, i, sum = 0;
//     cout << "Enter number: ";
//     cin >> n;
//     for (i = 1; i <= n; i++)
//     {
//         sum += i;
//     }
//     cout << "The Sum of " << n << " numbers is " << sum;
//     return 0;
// }

// Sum of squares:
// #include <iostream>
// using namespace std;
// int main()
// {
//     int n, i, sum = 0;
//     cout << "Enter number: ";
//     cin >> n;

//     for (i = 0; i <= n; i++)
//     {
//         sum += i * i;
//         cout << "\nsquare of " << i << " = " << i * i << "and sum = " << sum;
//     }
//     cout << "\nsum of square of " << n << " numbers square =" << sum;
//     return 0;
// }

//--------------------------------------------
// Sum of cubes:
#include <iostream>
using namespace std;
int main()
{
    int n, i, sum = 0;
    cout << "Enter number: ";
    cin >> n;

    for (i = 0; i <= n; i++)
    {
        sum += i * i * i;
        cout << "\ncube of " << i << " = " << i * i * i << " and sum = " << sum;
    }
    cout << "\nsum of cube of " << n << " numbers cube =" << sum;
    return 0;
}